### ISSUE TITLE
meta: project status

### ISSUE THREAD

**Issue by jo3-l** [yagpdb-cc/yagpdb-cc#86]
# Current project status

Hey all!

I'd just like to provide a notice about the project status currently - there's several open issues of note and also several PRs that haven't been reviewed. I apologize for this, I've been quite busy with school lately and most of my free time is directed towards my own private bot which I'm writing for a server I own.

I'm still going through the PRs regularly, so you can expect bugfixes and other small additions / changes to be merged quickly, though don't expect any large PRs to be merged until sometime later.

At this point in time I think I'll have some time to address the issues and review the open large PRs in this repo sometime later this month / early next month, no promises though!

## TL;DR
I'm busy with other things right now, but am still reviewing small PRs and issues regularly and plan to come back to the repo whenever I have time.

**Comment by Olde7325** [comment]
Hi, sorry to necro this!
Why not consider delegating a few (trusted) people to support you?
If you did - nevermind then 😅

**Comment by jo3-l2** [comment]
Nothing's really stopping that, really -
* Satty, Pedro, and Crafter already have collaborator access on this repository
* I've said that admins on the support server can contact me and I can give them owner access to the organization
* If VS are active on this repo (helping out in issues, submitting PRs, etc) then they can contact me and I can add them as a member
* It's not just VS/admins, normal users can get access too (obviously, there's a slightly higher entry point for normal users vs staff who have shown they can be trusted)

TL;DR: if you've shown activity on this repo and are interested in becoming a member, send a DM on Discord and I'll talk with the other maintainers about it.

**Comment by Olde7325** [comment]
Thanks for the insights!
I will definitely look forward to contribute more to this repository 👍
Have a happy new year!
